pkgs/top-level/all-packages.nix

```diff
@@ -21135,6 +21138,7 @@ with pkgs;
     else if name == "nblibc" then targetPackages.netbsdCross.libc or netbsdCross.libc
     else if name == "wasilibc" then targetPackages.wasilibc or wasilibc
     else if name == "relibc" then targetPackages.relibc or relibc
+    else if stdenv.targetPlatform.isEmscripten then null
```

Comment:
why is emscripten special from all the others, why is `stdenv.targetPlatform.libc` not null on emscripten

Reply:
I'll remove this and set `libc = null`